Test Failure: org.eclipse.microprofile.metrics.tck.TimerTest.testRate

RTC 268568

```
testRate:java.lang.AssertionError: expected:<1.0> but was:<1.1203899899654197>
at org.junit.Assert.fail(Assert.java:93)
at org.junit.Assert.failNotEquals(Assert.java:647)
at org.junit.Assert.assertEquals(Assert.java:443)
at org.junit.Assert.assertEquals(Assert.java:512)
at org.eclipse.microprofile.metrics.tck.TestUtils.assertEqualsWithTolerance(TestUtils.java:41)
at org.eclipse.microprofile.metrics.tck.TimerTest.testRate(TimerTest.java:107)
```

It should be fixed with https://github.com/OpenLiberty/open-liberty/pull/14840 but not sure why it happened again.

--- a/dev/com.ibm.ws.microprofile.metrics.1.1_fat_tck/publish/tckRunner/tck/pom.xml
+++ b/dev/com.ibm.ws.microprofile.metrics.1.1_fat_tck/publish/tckRunner/tck/pom.xml
@@ -16,7 +16,7 @@
     <name>MicroProfile Metrics TCK Runner 1.1 TCK Module</name>
 
     <properties>
-        <microprofile.metrics.version>1.1.1</microprofile.metrics.version>
+        <microprofile.metrics.version>1.1.3</microprofile.metrics.version>
         <arquillian.version>1.1.13.Final</arquillian.version>
 
         <!-- the below is used in arquillian.xml -->

--- a/dev/com.ibm.ws.microprofile.metrics.2.0_fat_tck/publish/tckRunner/tck/pom.xml
+++ b/dev/com.ibm.ws.microprofile.metrics.2.0_fat_tck/publish/tckRunner/tck/pom.xml
@@ -16,7 +16,7 @@
     <name>MicroProfile Metrics TCK Runner 2.0.1 TCK Module</name>
 
     <properties>
-        <microprofile.metrics.version>2.0.1</microprofile.metrics.version>
+        <microprofile.metrics.version>2.0.4</microprofile.metrics.version>
         <arquillian.version>1.1.13.Final</arquillian.version>
 
         <!-- the below is used in arquillian.xml -->

--- a/dev/com.ibm.ws.microprofile.metrics.2.2_fat_tck/publish/tckRunner/tck/pom.xml
+++ b/dev/com.ibm.ws.microprofile.metrics.2.2_fat_tck/publish/tckRunner/tck/pom.xml
@@ -16,7 +16,7 @@
     <name>MicroProfile Metrics TCK Runner 2.2 TCK Module</name>
 
     <properties>
-        <microprofile.metrics.version>2.2.2</microprofile.metrics.version>
+        <microprofile.metrics.version>2.2.3</microprofile.metrics.version>
         <arquillian.version>1.1.13.Final</arquillian.version>
 
         <!-- the below is used in arquillian.xml -->

--- a/dev/com.ibm.ws.microprofile.metrics.2.3_fat_tck/publish/tckRunner/tck/pom.xml
+++ b/dev/com.ibm.ws.microprofile.metrics.2.3_fat_tck/publish/tckRunner/tck/pom.xml
@@ -16,7 +16,7 @@
     <name>MicroProfile Metrics TCK Runner 2.3 TCK Module</name>
 
     <properties>
-        <microprofile.metrics.version>2.3.2</microprofile.metrics.version>
+        <microprofile.metrics.version>2.3.3</microprofile.metrics.version>
         <arquillian.version>1.1.13.Final</arquillian.version>
 
         <!-- the below is used in arquillian.xml -->

